README-dev-on-docker.md

```diff
@@ -0,0 +1,114 @@
+# Developing Vespa on Docker
+
+The purpose of this document is to describe how to set up a combined development and
+run-time environment for (iteratively) developing and testing Vespa.
+
+### Start the docker container
+
+    DOCKER_IMAGE=vespaengine/vespa-dev:latest
+    SOURCE_ROOT=${HOME}/vespa/vespa
+    APP_ROOT=${HOME}/vespa/sample-apps
+
+    docker run -d -p 8080:8080 -p 8998:8998 -p 10001:10001 \
```

Comment:
You should add `docker pull ${DOCKER_IMAGE} ` to ensure that `docker run` uses newest version of the image.